src/components/notification/store/sagas.ts

```diff
@@ -515,6 +530,7 @@ export function* getNotifications(isFirstFetch: boolean) {
               hasMore
             )
           )
+          yield processNewNotifications(notificationItems)
```

Comment:
My only thing is maybe rename this to `handleNewNotifications` bc `process` makes it seem like we are manipulating the notif data in some way